src/components/DatePicker/DatePicker.tsx

```diff
@@ -84,32 +82,22 @@ export class DatePicker extends React.PureComponent<CombinedProps, State> {
       disableDatesBefore,
       disableDatesAfter,
       weekStartsOn = Weekdays.Sunday,
-      polaris: {intl},
+      locale = 'en',
```

Comment:
Can't we get the locale at the provider level or something like that? Or maybe not?

Reply:
Also, this is more for the Polaris team to answer, but is this a pattern they want to have in their library for their components?